src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/CodeStyle/VisualBasicCodeStyleOptions.vb

```diff
@@ -53,10 +53,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
                 editorConfigName:="visual_basic_style_unused_value_assignment_preference",
                 defaultValue:=New CodeStyleOption2(Of UnusedValuePreference)(UnusedValuePreference.UnusedLocalVariable, NotificationOption2.Suggestion),
                 optionsBuilder:=s_allOptionsBuilder)
+
+        Public Shared ReadOnly ParenthesesForInvocations As [Option2](Of CodeStyleOption2(Of Boolean)) = CreateOption(
+            VisualBasicCodeStyleOptionGroups.Parentheses, NameOf(ParenthesesForInvocations),
+            defaultValue:=CodeStyleOptions2.FalseWithSuggestionEnforcement, 'Suggestion or silent??
```

Comment:
silent.  and should be true by default.

Reply:
> should be true by default.

Shouldn't false make them disabled by default?
You stated `The options should be disabled by default.` in this comment.

https://github.com/dotnet/roslyn/issues/45170#issuecomment-644392752

Or I misunderstand something?

Reply:
it shoudl be disabled.  i.e. 'silent'.  but the default *direction* of each option is different.  i.e. for methods, we default to sayign that they shoudl have parens (silently), and for properties we default to saying that they shoudl not have parens (silently).

Reply:
this optiosn can affect things like code-gen codepaths, where we might update them to respect the direction of these option, while teh severity indicates how we nag people about their existing code. :)